lib/Installation/NtpConfiguration/NtpConfigurationPage.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The module provides interface to act on the
+#          ntp Configuration page.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Installation::NtpConfiguration::NtpConfigurationPage;
+use parent 'Installation::Navigation::NavigationBase';
```

Comment:
we are avoiding to use NavigationBase, this was a bad choice in the past, please repeat the next method for now in this class.